packages/utils/telemetry-utils/src/test/errorLogging.spec.ts

```diff
@@ -620,7 +620,7 @@ describe("normalizeError", () => {
             assert.equal(actual.message, expected.message, "message should match");
             const actualStack = actual.stack;
             assert(actualStack !== undefined, "stack should be present as a string");
-            if (actualStack.indexOf("at Object.normalizeError") >= 0) { // This indicates the stack was populated naturally by new SimpleFluidError
+            if (actualStack.indexOf("at normalizeError") >= 0) { // This indicates the stack was populated naturally by new SimpleFluidError
```

Comment:
Is this an accidental merge artifact?

Reply:
This is a legitime change. The printed call stack is slightly different with the new typescript. I believe this is due to typescript fixing the bug that imported functions from modules were called in a way that set the 'this' parameter to the module. With this fix imported functions like normalizeError are no longer invokes as methods, and are instead properly invoked as functions, resulting in this stack change.

Reply:
Without this change, this test failed since its very picky about the exact string in printed call stacks.

Reply:
Makes sense.  Thanks!

Reply:
Thank you for fixing my picky test!  :P